hyperspy/io.py

```diff
@@ -339,6 +340,50 @@ def load_with_reader(filename,
     return objects
 
 
+def oneview_is_reader(folder_path):
+    """
+    Imports OneView_IS datasets.
+    These datasets are saved as nested folders with the format:
+    'Dataset/Hour_00/Minute_00/Second_00/*.dm4' etc.
+    Function parameters:
+     
+    folder_path (string): the directory path of the Dataset folder
+    ----------
+    Retuns:
+    imageStack: Hypespy Signal2D lazy stack of the frames, ordered sequentially
+                    The image axes parameters are copied over from first frame.
+                    The thrid axis is defines as 't', scaled with the camera
+                    exposure time, read from the first frame metadata.
+                    OneView_IS mode captures the frames continuously.
+    """
+     
+    os.chdir(folder_path)
+     
+    filenames_sort = []
+    for root, dirs, files in os.walk(folder_path):
+        for file in files:
+            if file.endswith('.dm4') or file.endswith('.dm3'):
+                filenames_sort.append(os.path.join(root, file))
+    
+    filenames_sort.sort()
+
+    #This works due to the naming format of these datasets.
+    #May need changing if to cover other formats
+     
+    imageStack = load(filenames_sort, lazy = True, stack = True)
```

Comment:
`lazy=True` should be an option with a default to `True` and not hard coded, to allow the user to load the data to memory.
Also since the default in hyperspy is to load the data to memory, I would suggest to inform the user using the logger at the info level (`_logger.info()`).